src/wavesurfer.ts

```diff
@@ -266,6 +266,15 @@ class WaveSurfer extends Player<WaveSurferEvents> {
           this.emit('click', relativeX, relativeY)
         }
       }),
+       // Right click
+    this.renderer.on('contextmenu', (relativeX, relativeY, event: MouseEvent) => {
+      event.preventDefault();  // Prevent default context menu
+      if (this.options.interact && event.button === 2) {  // Right-click
+        this.seekTo(relativeX)
```

Comment:
Why do we want to seek on `rightclick`?

Reply:
currently when we right click in portal we show the popup to add dialogue at the current time stamp. so if we do this we would be moving the cursor and then add dialogue at that point.

what we have rn adds dialogue where the waveform cursor is at which is not ideal.